doc/integrating/integration.md

```diff
@@ -25,9 +30,23 @@ app.mount(document.body);
 
 Spreadsheet component takes the following props:
 
+**required**:
+
 - `model`
   The spreadsheet model to be used with the component.
 
+**optional**:
+
+- `notifyUser`
+  A function used to notify the user. It supports several levels of severity as well as a sticky behaviour.
```

Comment:
seems strange to say the it supports "several levels of severity as well as a sticky behaviour" since the default one only does a window.alert with no nuanced behaviour.

Reply:
the API has to support it.